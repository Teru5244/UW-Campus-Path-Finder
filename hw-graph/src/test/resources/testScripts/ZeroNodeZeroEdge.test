# Test whether a graph can be generated

# Create a graph
CreateGraph g

# Print the nodes in the graph
ListNodes g