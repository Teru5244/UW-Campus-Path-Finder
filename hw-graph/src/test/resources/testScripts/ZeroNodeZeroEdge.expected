# Test whether a graph can be generated

# Create a graph
created graph g

# Print the nodes in the graph
g contains: